time stamp
int32 control_mode
float64 battery_voltage
bool light_control_enabled

light_state front_light_state
light_state rear_light_state

int32 motor_num
motor_state[] motors

motion_state motion
